src/cli/check.rs

```diff
@@ -112,10 +120,11 @@ impl ExecuteSubcommand for Command {
         // FILES
         for filename in self.filenames.iter() {
             let text = std::fs::read_to_string(filename)?;
+            let split_pattern = self.split_pattern.clone();
```

Comment:
Is the clone necessary?

Reply:
No it is not, sorry, must've been leftover from something I was trying. I'll remove that line.